openassessment/templates/openassessmentblock/response/oa_response.html

```diff
@@ -44,7 +44,7 @@ <h4 class="step__title">
                 <p>
                     {% trans "Enter your response to the question." %}
                     {% if submission_due %}
-                    {% trans "You can save your progress and return to complete your response at any time before the due date" %} (<span class="step__deadline"><span class="date">{{ submission_due|utc|date:"l, N j, Y H:i e" }}</span></span>).
+                    {% trans "You can save your progress and return to complete your response at any time before the due date" %} (<span class="step__deadline"><span class="date">{{ submission_due|timezone:time_zone|date:"l, N j, Y H:i e" }}</span></span>).
```

Comment:
Found this one by grepping for `step__deadline` in the project.

Before (against master):
<img width="856" alt="screen shot 2016-12-12 at 3 33 08 pm" src="https://cloud.githubusercontent.com/assets/7373924/21115762/c7670a52-c080-11e6-8f17-51a18f7dda1b.png">

After (this change):
<img width="865" alt="screen shot 2016-12-12 at 3 27 10 pm" src="https://cloud.githubusercontent.com/assets/7373924/21115781/d6a811dc-c080-11e6-9097-97a7eb5f3433.png">